.github/ISSUE_TEMPLATE/new_feature.md

```diff
@@ -1,28 +1,33 @@
 ---
 name: New feature
 about: Add a new feature
-title: "[Feature] Feature title"
+title: '[Feature] Feature title'
 labels: Feature
 assignees: ''
-
 ---
 
 ## Origin
-_Links to received feedback, user research or other findings._
-
 
+_Links to received feedback, user research or other findings._
 
 ## Motivation
-_A short description of what user needs or business goals this feature will solve._
-
 
+_A short description of what user needs or business goals this feature will solve._
 
 ## Hypotheses and assumptions
-_A list of hypotheses and assumptions we have made about the user or the proposed solution._
-
 
+_A list of hypotheses and assumptions we have made about the user or the proposed solution._
 
 ## Proposed solution
+
 _A coarse description of a proposed solution, that may include wireframes and graphic design._
 
+## Acceptance Criteria
+
+_List of relevant acceptance criteria as a part of a QA flow_
 
+- [] Renders colors as expected in dark and light mode.
+- [] Is translatable to English and Norwegian.
+- [] Does not drain battery
+- [] Works in up to 200% font size
+- [] Is navigatable
```

Comment:
+ Operable and perceivable using screen reader?

Reply:
🤦 I just found these changes as uncommited changes in my terminal and seems as I just stopped midway.... Thanks 😅 It was supposed to be "is navigatable using screen reader", but I like your wording better.